starboard/build/config/win/BUILD.gn

```diff
@@ -234,5 +231,10 @@ config("common") {
 
     # Deprecated function warning.
     "/wd4996",
+
```

Comment:
extra newline? Or should the warnings below have a comment of their own?

Reply:
Added some comments and tried to group them as a section of warnings added due to VS 2022. I think they should eventually be looked at and corrected, but for now I just have them here.